### PR TITLE
Document tauri.conf.json version default

### PR DESCRIPTION
**Pull request by probablykasper**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [ ] No

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Review by wusyong — APPROVED** [review]

**Comment by wusyong** [comment]
Could you open it toward `next` branch?
It's LGTM btw.

File: docs/api/config.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ In addition to the JSON defined on the `tauri.conf.json` file, Tauri reads a pla
 
 <Properties anchorRoot="package" rows={[
   { property: "productName", optional: true, type: "string", description: `Application name. The binary name is converted to snake-case on Linux.` },
-  { property: "version", optional: true, type: "string", description: `Application version.` }
+  { property: "version", optional: true, type: "string", description: `Application version. Defaults to the version specified in Cargo.toml` }
```
Comment (FabianLars) [inline docs/api/config.md:49]: Should we add the same note to the `productName` property?

**Comment by probablykasper** [comment]
@wusyong Opened #2550 to the `next` branch.

File: docs/api/config.md (hunk 1 above)
Comment (probablykasper) [reply docs/api/config.md:49]: Good idea, doing that

**Pull request closed without merging**